community/modules/scheduler/schedmd-slurm-gcp-v6-controller/controller.tf

```diff
@@ -0,0 +1,109 @@
+  source = "github.com/SchedMD/slurm-gcp.git//terraform/slurm_cluster/modules/slurm_instance_template?ref=6.2.0"
+  count  = local.have_template ? 0 : 1
+
+  project_id          = var.project_id
+  region              = var.region
+  slurm_instance_role = "controller"
+  slurm_cluster_name  = local.slurm_cluster_name
+  labels              = local.labels
+
+  disk_auto_delete = var.disk_auto_delete
+  disk_labels      = merge(var.disk_labels, local.labels)
+  disk_size_gb     = var.disk_size_gb
+  disk_type        = var.disk_type
+  additional_disks = local.additional_disks
+
+  bandwidth_tier    = var.bandwidth_tier
+  slurm_bucket_path = module.slurm_files.slurm_bucket_path
+  can_ip_forward    = var.can_ip_forward
+  disable_smt       = var.disable_smt
+
+  enable_confidential_vm   = var.enable_confidential_vm
+  enable_oslogin           = var.enable_oslogin
+  enable_shielded_vm       = var.enable_shielded_vm
+  shielded_instance_config = var.shielded_instance_config
+
+  gpu = one(local.guest_accelerator)
+
+  machine_type     = var.machine_type
+  metadata         = var.metadata
+  min_cpu_platform = var.min_cpu_platform
+
+  # TODO: add support for network_ip
+  # network_ip               = var.controller_instance_config.network_ip != null ? var.controller_instance_config.network_ip : ""
+  on_host_maintenance = var.on_host_maintenance
+  preemptible         = var.preemptible
+
+  service_account = coalesce(var.service_account, {
+    email  = data.google_compute_default_service_account.default.email
+    scopes = ["https://www.googleapis.com/auth/cloud-platform"]
+  })
+
+  source_image_family  = local.source_image_family             # requires source_image_logic.tf
+  source_image_project = local.source_image_project_normalized # requires source_image_logic.tf
+  source_image         = local.source_image                    # requires source_image_logic.tf
+
+  # TODO: add support for spot (?)
+  # spot                     = var.controller_instance_config.spot
+  subnetwork_project = var.subnetwork_project
+  subnetwork         = var.subnetwork_self_link
+
+  tags = concat([local.slurm_cluster_name], var.tags)
+  # TODO: add support for termination_action (?)
+  # termination_action       = var.controller_instance_config.termination_action
+}
+
+# INSTANCE
+module "slurm_controller_instance" {
+  source = "github.com/SchedMD/slurm-gcp.git//terraform/slurm_cluster/modules/slurm_controller_instance?ref=6.2.0"
+
+  slurm_cluster_name = local.slurm_cluster_name
+  project_id         = var.project_id
+  region             = var.region
+  zone               = var.zone
+  # TODO: add supoort for cloudsql
+  # cloudsql           = var.cloudsql
+
+  instance_template = local.have_template ? var.instance_template : module.slurm_controller_template[0].self_link
+
+  enable_public_ip = !var.disable_controller_public_ips
+  # TODO: add support for network_tier
+  # network_tier       = var.controller_instance_config.network_tier
```

Comment:
Can we clean up the commented out lines. I am fine with the comment and even the left side of the `=`. The right side of the equals is referencing a var that does not exist in our code, so without more context it is quite confusing. I would update the comments to something like:

`# network_tier = TODO: add support for network_tier`

Reply:
done